contributing.rst

```diff
@@ -306,30 +306,30 @@ The rest of the documentation assumes you are in the directory for the repo (the
   - The ``clone`` command creates a new directory inside the current one. So you do not need to create a new `odk-docs` directory first.
   - As noted above, we recommend a master :file:`odk` directory that holds your virtualenv directory and your git repo in two separate subdirectories. So you would be in that master :file:`odk` directory when you clone down the repo.
   - Double check that right folders are in the right places
-  
+
   .. code-block:: none
 
     - odk/
       - odkenv/
       - docs/
-	
+
 .. _upstream-the-docs:
 
 Set the Upstream Remote
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When you clone down a repo, the local copy calls your GitHub copy ``origin``. You should also set ``upstream`` as the name of the original, main GitHub repo.
+When you clone down a repo, the local copy calls your GitHub copy ``origin``. You should also set ``upstream`` as the name of the original master branch of the main GitHub repo.
 
 
 .. code-block:: console
 
-  $ git remote add --track upstream https://github.com/opendatakit/docs.git  
+  $ git remote add --track master upstream https://github.com/opendatakit/docs.git
```

Comment:
I don't think this change is correct. It looks like you are adding a remote named master, not a remote named upstream

I do think we need to remove `--track` from this. 
I can't remember why it is here, and it isn't part of the git documentation...

Reply:
Yes, about this one, while I was setting up my local environment the first time I pasted this line in my terminal console I've got the following error:

usage: git remote add [<options>] <name> <url>

    -f, --fetch           fetch the remote branches
    --tags                import all tags and associated objects when fetching
                          or do not fetch any tag at all (--no-tags)
    -t, --track <branch>  branch(es) to track
    -m, --master <branch>
                          master branch
    --mirror[=<push|fetch>]
                          set up remote as a mirror to push to or fetch from

and I saw that the before the name of the remote, when using the `--track` option one must define which branch(es) should be tracked. At least that what I understood from this message.

I solved it in my local environment by substituting the `--track` for the `--fetch`, then I was able to get the entire repo.

But before altering this part of the documment I also tested with way (`--track` option + branch name before the upstream name) and it worked. 

So I opted for the edition that would change less of the documment that was adding the branch name after the `--track` instead of changing the in code line the `--track` to `--fetch`).

But if we need to remove the `--track` what shoud we replace it with?

Reply:
I think if the command was just `git add remote upstream`, everything would work just fine.
Specifying  a tracking branch doesn't really seem to matter, and it doesn't change anything else we ask people to do.

Reply:
Alright, I'll edit just the line of code to the one you suggested.